layouts/api/list.html

```diff
@@ -368,37 +369,82 @@ <h3 class="mb-2">Code Example</h3>
 
+                                    {{ if eq $file_extension "sh"}}
+                                    {{ else }}
                                     <li class="nav-item">
                                         <a class="nav-link mr-1 js-code-example-link text-capitalize {{ if eq $count 0}} active {{end}}" data-code-lang="{{ $lang.pygments_name }}"  href="#" data-toggle="tab">{{ $lang.pygments_name }}</a>
                                     </li>
 
                                     {{ $count = add $count 1 }}
+                                    {{ end }}
 
                                     {{ end }}
 
                                 </ul>
 
-                                {{ $count := 0}}
+                                {{ $count := 0 }}
+                                {{ $file_extension := "sh" }}
+                                {{ $lang := index $dot.Site.Data.highlighting.languages "sh" }}
+                                <div class="code-snippet js-code-block {{if eq $file_extension "sh"}} default {{end }} {{ if eq $count 0}}d-block{{else}}d-none{{end}} code-block-{{ $lang.pygments_name }}" >
+                                    <div class="code-button-wrapper position-absolute">
+                                        <button class="btn text-primary js-copy-button">Copy</button>
+                                    </div>
+                                    {{ $pathParams := $.Scratch.Get "pathParams" }}
+                                    <div class="highlight">
+                                        <pre class="chroma">
+                                            <code class="language-fallback" data-lang="fallback">
+{{- with $pathParams -}}
+{{- range . -}}
+export {{ .name }}="{{ .example | default "CHANGE_ME"}}"
+{{ end -}}
+{{- end -}}
+{{ if $context.action.requestBody -}}
+(cat << EOF
+{{ ($.Scratch.Get "jsonRequestBody") }}
+EOF
+) | curl -X <span class="text-uppercase">{{ $context.actionType }}</span> \
+-d @- \
+{{- else -}}
+curl -X <span class="text-uppercase">{{ $context.actionType }}</span> \
+{{- end }}
+-H "DD-API-KEY: ${DD_API_KEY}" \
+-H "DD-APPLICATION-KEY: ${DD_APP_KEY}" \
+{{ range $.Scratch.Get "serverURLS" }}<span class="d-none" data-region="{{ .region }}">{{ .url }}</span>{{ end }}<span>{{ replace $context.pathKey "{" "${" }}</span></code>
+                                        </pre>
+                                    </div>
+                                </div>
+
+                                {{ $count := 1 }}
                                 {{ range . }}
 
                                     {{ $file_content := .Content }}
                                     {{ $file_extension := index (split (path.Ext .) ".") 1 }}
                                     {{ $lang := index $dot.Site.Data.highlighting.languages $file_extension }}
-                            
+
+                                    {{ if eq $file_extension "sh"}}
+                                    {{ else }}
```

Comment:
You can probably change these if there is no logic for here
```suggestion
                                    {{ if ne $file_extension "sh"}}
```